### PR TITLE
updated cssgrids calculator, updated flexbox calculator, created nasa…

### PR DESCRIPTION
**Pull request by petegilk**
… project directory

Finished my CSS grid calculator, still working on flex but i thought i'd go ahead and turn it in

### CODE REVIEW

File: src/code-a-long/calculator/calc-grid.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Document</title>
```
Comment (juan-rod) [inline src/code-a-long/calculator/calc-grid.html:7]: update title name to describe what the project is

Comment (petegilk) [reply src/code-a-long/calculator/calc-grid.html:7]: Thanks, will do

**Review by juan-rod — COMMENTED** [review]
really good so far. very clean. and I like the styling. takes me back to the first version of iOS skeuomorphic design. word is it's coming back in style!

I'm assuming the nasa project will be on this branch so I'll wait for that before giving merge approval.

File: src/code-a-long/calculator/calc-grid.html (hunk 1 above)
Comment (petegilk) [reply src/code-a-long/calculator/calc-grid.html:7]: Ok, I added my NASA page. I'm not sure if I need to make another pull request, but let me know if I do

**Pull request merged**